### ISSUE TITLE
[0.26.1] Error syncing to wandb

### ISSUE THREAD

**Issue by thehappydinoa** [openai/openai-python#189]
When running the following command I receive the following warning.

```sh
$ openai wandb sync --project project
wandb: Currently logged in as: me (company). Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.13.9
wandb: Run data is saved locally in project/wandb/run-20230117_161328-ft-upiwOdAB20VtkGN7uHW5TFv2
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ft-upiwOdAB20VtkGN7uHW5TFv2
wandb: ⭐️ View project at https://wandb.ai/me/project
wandb: 🚀 View run at https://wandb.ai/me/project/runs/ft-upiwOdAB20VtkGN7uHW5TFv2
File file-ISyZKcogrCzx59ftjGPWOUs1 could not be read as a valid JSON file
File file-3GjQ844dqjAY6hYuQKpMf7mQ could not be read as a valid JSON file
```

When I check in wandb, only system metrics are synced. I do not see any of the metrics related to the fine-tuning such as validation loss and token usage.

I am running the following versions.

```
python==3.10.8
openai==0.26.1
wandb==0.13.9
```

**Comment by borisdayma** [comment]
Hi,

Can you try to log only latest fine-tune run to see if it's an issue with this specific one:
```bash
openai wandb sync -n 1 --project my_project
```

**Comment by thehappydinoa** [comment]
Hey @borisdayma , I tried it with the last one and the last two with the same error.

**Comment by hallacy** [comment]
0.26.3 just got released if you'd like to try it out for this issue
